Document/0x05a-Platform-Overview.md

```diff
@@ -48,6 +48,37 @@ For example, Android Nougat defines the following system users:
     ...
 ```
 
+#### Android Device Encryption
+
+//INTRO
+
+##### Full-Disk Encryption
+
+//Full-Disk Encryption content
+
+##### File-Based Encryption
+
+//File-Based Encryption content
+
+##### Adiantum 
+
+AES is used on most modern Android devices for storage encryption. Actually, AES has become such a widely used algorithm that the most recent processor implementations have a dedicated set of instructions to provide hardware accelerated encryption and decryption operations, such as ARMv8 with its Cryptography Extensions or x86 with AES-NI extension.
+However, not all devices are capable of using AES for storage encryption in a timely fashion. Especially those that use Android Go (an Android version targeted at low-end devices), smartwatches or TV's. These devices usually use low-end processors, such as the ARM Cortex-A7 which don't have hardware accelerated AES.

```

Comment:
Can you change `Especially those that use Android Go (an Android version targeted at low-end devices), ` to `Especially low-end devices running Android Go.`